### PR TITLE
add Google Analytics

### PR DESCRIPTION
**Pull request by dmitrytrager**
related to #301

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,13 +2,14 @@
 
 ## Unreleased
 
-## 1.3 - 2015-07-03
+## 1.3 - 2015-07-27
```
Comment (timurvafin) [inline CHANGELOG.md]: No need to change release date, just add your line in the "Unreleased" section.

**Comment by timurvafin** [comment]
Please mention env variable in the .env.example with descriptive comment above.

**Comment by dmitrytrager** [comment]
updated

**Comment by timurvafin** [comment]
@dmitrytrager please make sure your branch is could be merged without conflicts.

**Comment by dmitrytrager** [comment]
@timurvafin done

**Pull request merged**